ui/index.js

```diff
@@ -1,47 +1,15 @@
 import React from "react";
 import ReactDOM from "react-dom";
-import {
-    Masthead, Nav, NavItem, TabContent, TabPane, TabContainer
-} from "patternfly-react";
 import ApolloClient from "apollo-boost";
 import { ApolloProvider } from "react-apollo";
-import DataSourcesView from "./components/DataSourcesView";
 
-const App = () => {
-    const client = new ApolloClient({ uru: "/graphql" });
+import App from "./components/app";
 
-    return (
-        <ApolloProvider client={client}>
-            <div>
-                <Masthead
-                    iconImg="img/logo.png"
-                    title="AeroGear Sync"
-                    navToggle={false}
-                />
-                <div>
-                    <div className="ag-header">
-                        <p>My Data Sync API</p>
-                    </div>
-                    <TabContainer id="basic-tabs">
-                        <div>
-                            <Nav bsClass="nav nav-tabs">
-                                <NavItem disabled={false}>
-                                    <div>Data Sources</div>
-                                </NavItem>
-                            </Nav>
-                            <TabContent>
-                                <TabPane>
-                                    <DataSourcesView />
-                                </TabPane>
+const client = new ApolloClient();
```

Comment:
Is `/graphql` the default uri? No other config required?

Reply:
That is correct https://www.apollographql.com/docs/react/essentials/get-started.html#creating-client